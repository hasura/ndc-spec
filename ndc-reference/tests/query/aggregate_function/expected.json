[
  {
    "aggregates": {
      "min_id": 1,
      "max_id": 3
    }
  }
]