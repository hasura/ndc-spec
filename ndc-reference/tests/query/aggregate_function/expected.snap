---
source: ndc-reference/bin/reference/main.rs
expression: response.0
input_file: ndc-reference/tests/query/aggregate_function/request.json
---
[
  {
    "aggregates": {
      "min_id": 1,
      "max_id": 3
    }
  }
]
